src/pages/fr/guides/publish-to-npm.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Publiez vers NPM
+description: Apprennez comment publier des composants Astro vers NPM
+---
+
+Vous développez un nouveau composant Astro ? **Publiez-le sur [NPM](https://npmjs.com/)** !
+
+Publier un composant Astro est une excellente façon de réutiliser votre travail existant dans vos projets, et de le partager avec la communauté d'Astro à grande échelle. Les composants Astro peuvent être publiés directement et installés depuis NPM, comme tout autre Package JavaScript.
+
+Vous cherchez de l'inspiration ? Regardez quelques-uns de nos [thèmes](https://astro.build/themes/) et [composants](https://astro.build/integrations/) de la communauté. Vous pouvez aussi [rechercher sur NPM](https://www.npmjs.com/search?q=keywords:astro-component) pour voir l'ensemble du catalogue public.
```

Comment:
Besoin d'inspiration ? Jetez un oeil à nos [thèmes](https://astro.build/themes/) et [composants](https://astro.build/integrations/) favoris. Vous pouvez aussi [rechercher sur NPM](https://www.npmjs.com/search?q=keywords:astro-component) pour voir l'ensemble du catalogue public.

Reply:
C'est pas les thèmes et composants aimés par Astro, mais la liste des intégrations et thèmes de la communauté sans distinction

Reply:
`Looking for inspiration? Check out some of our favorite [themes](https://astro.build/themes/) and [components](https://astro.build/integrations/) from the Astro community. You can also [search npm](https://www.npmjs.com/search?q=keywords:astro-component) to see the entire public catalog.` 
C'est un mélange des deux (favoris et de la commu) mais ça sonne un peu bizarre "nos thèmes favoris de la communauté" non? Il faudrait une autre tournure de phrase sans doute :thinking:

Reply:
Que pense tu de ça ?
`
Vous cherchez de l'inspiration ? Regardez quelques-uns de nos [thèmes](https://astro.build/themes/) et [composants](https://astro.build/integrations/) préférés offert par la communauté d'Astro. Vous pouvez aussi [rechercher sur NPM](https://www.npmjs.com/search?q=keywords:astro-component) pour voir l'ensemble du catalogue public.
`

Reply:
Oui, ça ou sinon ça :arrow_down: 
` Vous cherchez de l'inspiration ? Regardez quelques-uns de nos [thèmes](https://astro.build/themes/) et [composants](https://astro.build/integrations/) préférés issus la communauté d'Astro. Vous pouvez aussi [rechercher sur NPM](https://www.npmjs.com/search?q=keywords:astro-component) pour voir l'ensemble du catalogue public.`

Reply:
On pars la dessus alors !